_episodes/03-github.md

```diff
@@ -7,16 +7,18 @@ questions:
 objectives:
 - "Explain reasons to use GitHub."
 
-keypoints:
+key points:
```

Comment:
This is used in the template to render the green box of "Key Points" at the end of each lesson. Adding a space here may make this section not recognized by the template, so it should likely be switched back to "keypoints"

Reply:
Ah. Thanks. I will refrain from editing those blocks.